crown/src/utils/bytes.rs

```diff
@@ -99,6 +99,12 @@ impl ToBytes for &[u8] {
     }
 }
 
+impl ToBytes for Vec<u8> {
+    fn to_bytes(&self) -> Result<Vec<u8>> {
+        Ok(self.clone())
```

Comment:
Does this need to be `.clone()`?

Reply:
Can't return self, compiler told me to use clone. Perhaps could do something with a mutable reference instead to take ownership.

Reply:
Oh I see it's because the trait prototype is a reference. Ugh, that's fine I guess.